Basic/Binding/Basic/Http/CS/Service/Properties/launchSettings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true
+  },
+  "profiles": {
+    "Service": {
```

Comment:
I would expect this node to look something like this to enable Kestrel:
```
    "Service": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "applicationUrl": "[http://localhost:5001](http://localhost:5001)",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
```
As Kestrel will launch on port 5001, you need to do something on the client side to override the Url used, and maybe choose via a command line switch.

Reply:
Added missing parts of the config, now the service works on Kestrel/IIS/Express